src/phare/phare.hpp

```diff
@@ -32,7 +32,7 @@ class SamraiLifeCycle
         SAMRAI::tbox::SAMRAI_MPI::init(&argc, &argv);
         SAMRAI::tbox::SAMRAIManager::initialize();
         SAMRAI::tbox::SAMRAIManager::startup();
-
+        SAMRAI::tbox::SAMRAI_MPI::setCallAbortInParallelInsteadOfMPIAbort();
```

Comment:
to keep?
the other one for callign abort in serial ops is also useful for stacktraces

Reply:
do you remember why I added this? I know we discussed it but can't remember...

Reply:
probably to get a stack trace

Reply:
I commented it